### PR TITLE
Fix bug for US Sen Tammy Duckworth Yaml (VAN-50169)

### PR DESCRIPTION
**Pull request by OdeclasV**
The previous commit I did had the wrong steps from topic and state variables. This fixes those issues. I tested they yaml on the Tester and everything worked properly.

### CODE REVIEW

File: members/D000622.yaml [hunk 1]
```diff
@@ -25,6 +25,17 @@ contact_form:
             Sister: Sister
             Rabbi: Rabbi
             Reverend: Reverend
+        - name: topic
+          selector: select#input-356E5B93-5056-A066-6052-9F850C8A291E
+          value: $TOPIC
+          required: true
+          options:
+            Agriculture, Farm Bill: "Agriculture, Farm Bill"
```
Comment (annanhan) [inline members/D000622.yaml]: are these the only topic options on her form?

Comment (OdeclasV) [reply members/D000622.yaml]: Right. Those are not the only ones. Adding them now

File: members/D000622.yaml [hunk 2]
```diff
@@ -25,6 +25,17 @@ contact_form:
...
+            Animals: "Animals"
+        - name: state
+          selector: select#input-356E5B73-5056-A066-60BE-0B9B6F422B57
+          value: $ADDRESS_STATE_POSTAL_ABBREV
+          required: true
```
Comment (tomdionesotes) [inline members/D000622.yaml]: It will be helpful for our yaml testing if you add IL as an option here (so it will prefill as that in the yaml tester)

**Comment by tomdionesotes** [comment]
lgtm

**Pull request merged**